### PR TITLE
Factor c

### PR DESCRIPTION
**Pull request by fieker**

### CODE REVIEW

File: test/NumField/NfAbs/ConjugatesNS.jl [hunk 1]
```diff
@@ -17,3 +17,18 @@
...
 end
+
+@testset "NumField/NfAbsNS/Conjugates.jl" begin
+  Qx, x = QQ["x"]
+  K, a = wildanger_field(3, 13)
+  r, s = @inferred signature(K)
+  @test (r, s) == (1, 1)
+
+  c = conjugates(b, ComplexField(64))
+  cc = conjugates(b, 64)
```
Comment (thofma) [inline test/NumField/NfAbs/ConjugatesNS.jl:28]:
```suggestion
  c = conjugates(a, ComplexField(64))
  cc = conjugates(a, 64)
```

File: test/NumField/NfAbs/ConjugatesNS.jl [hunk 2]
```diff
@@ -17,3 +17,18 @@
...
+
+@testset "NumField/NfAbsNS/Conjugates.jl" begin
+  Qx, x = QQ["x"]
+  K, a = wildanger_field(3, 13)
+  r, s = @inferred signature(K)
+  @test (r, s) == (1, 1)
+  b = rand(K, -1:1)
+
+  c = conjugates(a, ComplexField(64))
+  cc = conjugates(a, 64)
```
Comment (thofma) [inline test/NumField/NfAbs/ConjugatesNS.jl:29]:
```suggestion
  c = conjugates(b, ComplexField(64))
  cc = conjugates(b, 64)
```

**Comment by fieker** [comment]
thanks

**Comment by thofma** [comment]
Still some issues.

**Comment by fieker** [comment]
Yay!!!

**Pull request merged**